pkg/ccl/backupccl/restore_span_covering.go

```diff
@@ -674,18 +681,12 @@ func (i *fileSpanStartAndEndKeyIterator) next() {
 			break
```

Comment:
could you add a comment explaining why we would hit this break point? I'm unsure if we do anymore, now that we only yield start keys

Reply:
I see, thanks!